tests/plugins/test_build_source_container.py

```diff
@@ -80,10 +85,75 @@ def test_running_build(tmpdir, caplog):
             'args': {},
         }]
     )
-    build_result = runner.run()
-    assert not build_result.is_failed()
-    assert build_result.oci_image_path
-    assert 'stub stdout' in caplog.text
+
+    temp_source_dir = os.path.join(str(tmpdir), 'source_dir')
+    temp_image_output_dir = os.path.join(str(tmpdir), 'image_output_dir')
+    temp_image_export_dir = os.path.join(str(tmpdir), 'image_export_dir')
+    tempfile_chain = flexmock(tempfile).should_receive("mkdtemp").and_return(temp_source_dir)
+    tempfile_chain.and_return(temp_image_output_dir)
+    tempfile_chain.and_return(temp_image_export_dir)
```

Comment:
Minor: this depends on the order of calls in the implementation, but I don't see an easy alternative.

Reply:
yeah I know it does